be/src/agent/task_worker_pool.cpp

```diff
@@ -618,6 +619,116 @@ void* TaskWorkerPool::_push_worker_thread_callback(void* arg_this) {
 
 void* TaskWorkerPool::_publish_version_worker_thread_callback(void* arg_this) {
     TaskWorkerPool* worker_pool_this = (TaskWorkerPool*)arg_this;
+
+    std::unique_ptr<ThreadPool> threadpool;
+    auto st = ThreadPoolBuilder("publish_version")
+                      .set_min_threads(config::partition_publish_version_worker_count)
+                      .set_max_threads(config::partition_publish_version_worker_count)
+                      .set_max_queue_size(32)
```

Comment:
queue full may result in submit return ServiceUnavailable

Reply:
Updated. The status of `submit_func` is checked, and would retry task after a while.

Reply:
add some comment why use this magic number

Reply:
Updated.